WORKSPACE

```diff
@@ -50,9 +50,8 @@ new_http_archive(
   build_file = "compilers/gcc_arm_rpi.BUILD",
   strip_prefix = "tools-master/",
   # Prefer tar over ZIP, bazel complains about "Zip entries cannot refer to files outside of their directory: raspberry-tools-master.zip has a symlink to ../../lib/libBrokenLocale.so.1"
-  # url = "http://127.0.0.1:8000/raspberry-tools-master.tar.gz",
   url = "https://github.com/raspberrypi/tools/archive/master.tar.gz",
-  sha256 = "e34386e82819ac4c03ebcc4feddaa08e523cd562149379118270d99c1601d465"
+  sha256 = "eb2b2ea415b6894d571ac9c7622a320b3b0053a5166fb59821e62e9099429ac1"
```

Comment:
It is, there were few enough changes over the past months so that would not be a problem. We could pin a specific commit, however (there is no formal release as much as I remember).